nirc_ehr/resources/domain-templates/ehr.template.xml

```diff
@@ -0,0 +1,125 @@
+                <dat:column columnName="currentState">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="ownerId">
+                    <dat:rangeURI>int</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="userReferenceNumber">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isUnrestrictedView">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isAudit">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isAuditModified">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="protocolTypeId">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="parentProtocolId">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="questionnaireId">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="amendmentReason">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="protocolCategoryId">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="objective">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="rationale">
+                    <dat:rangeURI>string</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isTissueHarvest">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isOrderTracking">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isLongTerm">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isInVivo">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isStock">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+                <dat:column columnName="isHonorReset">
+                    <dat:rangeURI>boolean</dat:rangeURI>
+                </dat:column>
+            </dat:columns>
+        </table>
+    </template>
+
+    <template xsi:type="EHRTemplateType">
+        <table tableName="project" tableDbType="TABLE">
+            <dat:columns>
+                <dat:column columnName="isActive">
```

Comment:
Do we need this? Project already has a StartDate and EndDate column which are often enough to do a calculated isActive field.

Reply:
nope, removed the full defn as I don't need the other column as well.